ezgui/src/drawing.rs

```diff
@@ -227,6 +229,7 @@ pub struct Prerender {
     pub(crate) inner: PrerenderInnards,
     pub(crate) assets: Assets,
     pub(crate) num_uploads: Cell<usize>,
+    pub scale_factor: RefCell<f64>,
```

Comment:
`pub(crate)` too; I want to move away from exposing `Prerender` at all outside ezgui, so keeping things locked down here is important. (`Prerender` is only public at all right now to let client code upload geometry during `event` or `draw`.)

Reply:
done!